src/quoting-client.js

```diff
@@ -35,6 +36,7 @@ class QuotingClient {
       let query = {
         source_ledger: source,
         destination_ledger: destination,
+        destination_account: destinationAccount,
```

Comment:
Its just an incomplete payment object. It isn't _needed_ for pathfinding, its the result of pathfinding. `findPath` returns a list of payment template objects, which the sender fills in and uses.